styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    color: blue;
+}
```

Comment:
Hi @Sady-dotcom ,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    <img src="https://i.imgur.com/nvrw8p7.gif" alt="Remote Collaboration" style="width:100%;">
</div>

### Highlights
- ✅ **Good use of Gitflow**
- ✅ **Good use of HTML semantics**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._